Duplicate isReadOnly declaration blocking asset builds

## Description

The TriggerForm component has a duplicate `isReadOnly` variable declaration (lines 37 and 46) that is causing esbuild to fail, which blocks Phoenix from rebuilding JavaScript assets.

## Error

```
✘ [ERROR] The symbol "isReadOnly" has already been declared

    js/collaborative-editor/components/inspector/TriggerForm.tsx:46:10:
      46 │   const { isReadOnly } = useWorkflowReadOnly();
         ╵           ~~~~~~~~~~

  The symbol "isReadOnly" was originally declared here:

    js/collaborative-editor/components/inspector/TriggerForm.tsx:37:10:
      37 │   const { isReadOnly } = useWorkflowReadOnly();
         ╵           ~~~~~~~~~~
```

## Impact

- **Critical**: Completely blocks asset builds
- Phoenix cannot automatically rebuild JavaScript/TypeScript files
- All frontend development is blocked

## Solution

Remove the duplicate declaration on line 46.

## Steps to reproduce

1. Try to build assets: `mix esbuild default`
2. Build will fail with the duplicate declaration error

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -38,6 +38,8 @@ and this project adheres to
 
 ### Fixed
 
+- Duplicate `isReadOnly` declaration in TriggerForm that was blocking asset builds
+  [#3976](https://github.com/OpenFn/lightning/issues/3976)
 - Run duration and status alignment drift in history view
   [#3945](https://github.com/OpenFn/lightning/pull/3945)
 - Shared doc lookup in clustered environments now works across nodes instead of

--- a/assets/js/collaborative-editor/components/inspector/TriggerForm.tsx
+++ b/assets/js/collaborative-editor/components/inspector/TriggerForm.tsx
@@ -43,7 +43,6 @@ export function TriggerForm({ trigger }: TriggerFormProps) {
   const sessionContext = useSessionContext();
   const { provider } = useSession();
   const channel = provider?.channel;
-  const { isReadOnly } = useWorkflowReadOnly();
 
   // Get active trigger auth methods from workflow store
   const activeTriggerAuthMethods = useWorkflowState(

